app/main.js

```diff
@@ -41,7 +41,10 @@ const appDelegate = {
 
     log.info('Running version', version);
 
-    appDelegate._startBackend();
+    // Backend must be installed as a system service on windows
+    if (process.platform !== 'win32') {
```

Comment:
If there is an effort to make Linux service we should probably whitelist `linux` here as well.

Reply:
Technically, you should already be able to do so without any issues. On linux, our installers install and start the daemon as a service during the installation.